cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -46,23 +46,23 @@ $(SIM_BUILD)/$(TOPLEVEL): $(SIM_BUILD)/Vtop.mk
 	CPPFLAGS="$(SIM_BUILD_FLAGS)" make -C $(SIM_BUILD) -f Vtop.mk
```

Comment:
At this line, `make` can also be converted into `$(MAKE)` as done in the main [Makefile](https://github.com/cocotb/cocotb/blob/0652491035907568f2d37e99611559db3889f1b7/Makefile#L38=)

This would help with an unofficial BSD support: they provide `make` as `gmake`.

Should I add another branch for this?